.circleci/config.yml

```diff
@@ -31,6 +31,35 @@ jobs:
           paths:
             - plotly.js
 
+  timezone-jasmine:
+    docker:
+      # need '-browsers' version to test in real (xvfb-wrapped) browsers
+      - image: circleci/node:12.22.1-browsers
+    working_directory: ~/plotly.js
+    steps:
+      - attach_workspace:
+          at: ~/
+      - run:
+          name: Run hover_label test in UTC timezone
+          environment:
+            TZ: "UTC"
+          command: date && npm run test-jasmine hover_label
+      - run:
+          name: Run hover_label test in Europe/Berlin timezone
+          environment:
+            TZ: "Europe/Berlin"
+          command: date && npm run test-jasmine hover_label
+      - run:
+          name: Run hover_label test in Asia/Tokyo timezone
+          environment:
+            TZ: "Asia/Tokyo"
+          command: date && npm run test-jasmine hover_label
+      - run:
+          name: Run hover_label test in America/Toronto timezone
+          environment:
+            TZ: "America/Toronto"
```

Comment:
Do we need all these time zones to be tested?

Reply:
It's pretty quick, might as well keep them all for now.